routes/profileRoute.js

```diff
@@ -0,0 +1,17 @@
+const router = require('express').Router()
+const profileController = require('../controllers/profileController')
+const multer = require('multer')
+const upload = multer({ dest: 'uploads/'})
```

Comment:
my oga , let multer know that you want only png,jpg files
you can check the post route

Reply:
alright sure. thanks. I think I'll create a multer config file and then import it where multer is needed.